ComponentKit/Utilities/CKComponentAction.mm

```diff
@@ -249,3 +249,8 @@ void _CKTypedComponentDebugCheckTargetSelector(id target, SEL selector, const st
 }
 
 @end
+
+#pragma mark - Template instantiations
+
+template class CKTypedComponentAction<>;
+template class CKTypedComponentAction<id>;
```

Comment:
we declared extern in the header, we do have to actually do the instantiation somewhere...